pkg/ebpf/controlplane/controller.go

```diff
@@ -169,16 +222,298 @@ func (p *Controller) processCgroupMkdir(args []trace.Argument) error {
 	return nil
 }
 
-func (p *Controller) processCgroupRmdir(args []trace.Argument) error {
+func (ctrl *Controller) processCgroupRmdir(args []trace.Argument) error {
 	cgroupId, err := parse.ArgVal[uint64](args, "cgroup_id")
 	if err != nil {
-		return errfmt.Errorf("error parsing cgroup_rmdir args: %v", err)
+		return errfmt.Errorf("error parsing cgroup_rmdir signal args: %v", err)
 	}
-
 	hId, err := parse.ArgVal[uint32](args, "hierarchy_id")
 	if err != nil {
-		return errfmt.Errorf("error parsing cgroup_rmdir args: %v", err)
+		return errfmt.Errorf("error parsing cgroup_rmdir signal args: %v", err)
+	}
+	ctrl.cgroupManager.CgroupRemove(cgroupId, hId)
+
+	return nil
+}
+
+//
+// SchedProcessFork, SchedProcessExec and SchedProcessExit
+//
+
+const SchedProcessForkAmountOfArgs = 10
```

Comment:
Is this used anywhere outside this file? If not it should be internal to the process function (and if kept outside at least not capitalized as to not export it).